src/sources/readaheadframebuffer.h

```diff
@@ -92,17 +92,6 @@ class ReadAheadFrameBuffer final {
     bool tryContinueReadingFrom(
             FrameIndex readIndex);
 
-    enum class DiscontinuityOverlapMode {
-        Ignore,
-        Rewind,
-        Default = Rewind, // recommended default
-    };
-    enum class DiscontinuityGapMode {
-        Skip,
-        FillWithSilence,
-        Default = FillWithSilence, // recommended default
-    };
```

Comment:
I don't know why this was introduced in the first place but I assume there was a good reason for it. Is there a better reason for removing it or why did you choose to do so?

Reply:
I have removed it, because it was no longer used. This helped to have a better understanding how the code works and about the corner cases to consider.

Reply:
Okay going through the code again, I also don't see much value in keeping the `Skip` and `Ignore` modes.